requirements.txt

```diff
@@ -31,7 +31,7 @@ lxml==5.3.0
     #   eutils
     #   lxml-html-clean
     #   metapub
-lxml-html-clean==0.2.2
+lxml-html-clean==0.4.0
```

Comment:
can you remove this file. We have switched to tracking dependencies in pyproject.toml.